package.json

```
@@ -139,7 +141,8 @@
     "stylis-plugin-rtl-sc": "npm:stylis-plugin-rtl@1.1.0",
     "ts-jest": "29.1.0",
     "typescript": "5.0.4",
-    "yargs": "17.7.2"
+    "yargs": "17.7.2",
+    "yarn-deduplicate": "^6.0.1"
```

Pin the version and run `yarn` again
```suggestion
    "yarn-deduplicate": "6.0.1"
```

---

Thx